app/main/menu.ts

```diff
@@ -89,7 +89,7 @@ class AppMenu {
 			accelerator: process.platform === 'darwin' ? 'Alt+Command+I' : 'Ctrl+Shift+I',
 			click(_item: any, focusedWindow: any) {
 				if (focusedWindow) {
-					focusedWindow.webContents.toggleDevTools();
+					focusedWindow.webContents.openDevTools({mode: 'undocked'});
```

Comment:
This `undocked` thing open the devtools, right?

Reply:
Yes, it opens them in a new window, as we can't show them inside the window now.